### ISSUE TITLE
Compile time error on Linkage.swift on 2.12.0

### ISSUE THREAD

**Issue by peymankh** [yonaskolb/XcodeGen#766]
Getting a compile time error on Linkage.swift `defaultLinkage` property.

```
  Linkage.swift:12:9: note: add missing case: '.xcFramework'
        switch type {
        ^
```

**Comment by yonaskolb** [comment]
Fixed in 2.13.0

**Comment by peymankh** [comment]
Thanks for getting it sorted so quickly 👏
